client/flutter/lib/pages/symptom_checker/question_pages/short_list_question_view.dart

```diff
@@ -46,64 +48,110 @@ class _ShortListQuestionViewState extends State<ShortListQuestionView> {
 
   @override
   Widget build(BuildContext context) {
-    return SafeArea(
-      child: Padding(
-        padding: const EdgeInsets.all(32.0),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.center,
-          children: <Widget>[
-            Spacer(),
-            Container(
-                width: 300,
-                child: Html(data: widget.pageModel.question.questionHtml)),
-            SizedBox(height: 24),
-            ...widget.pageModel.question.answers.map(_buildAnswerRow).toList(),
-            Spacer(flex: 3),
-            PreviousNextButtons(
-                showPrevious: widget.pageModel.questionIndex > 0,
-                enableNext: _isComplete,
-                onPrevious: _previous,
-                onNext: _next),
-            Spacer()
-          ],
-        ),
-      ),
-    );
-  }
+    final answers = widget.pageModel.question.answers;
+    final group = _allowsMultipleSelection
+        ? PickOneOrMoreOptionGroup(
+            items: [
+              ...answers.map((a) => MultiSelectOptionItem(
+                    title: a.title,
+                    selected: _multipleSelections.contains(a.id),
+                  )),
+              // TODO: Localize
+              MultiSelectOptionItem(
```

Comment:
Do we want to default to having a "None of the above" option defined by default in code for all multi-select questions? I like the support you have for auto-deselecting other answers when it's chosen, but I worry that it will make the actual survey in the app less clearly predictable from the YAML, and removes the option for customizing the text of the "none of the above" option.

Maybe it's better long term to allow specifying a special key in the yaml to identify an answer option as a "none of the above" type that can use the auto-deselect logic you've created?

Reply:
Great idea!  But I won’t do it in this PR so we can get this in the build tonight.
In the future I’ll add the following as an answer:
```
- id: no_symptoms
  None_of_the_above: true
  Title: None of the above
```

Then we can use `has(‘no_symptoms’)` in a lot of conditions too!